js/adapt-languagePicker.js

```diff
@@ -38,7 +38,13 @@ define([
    * If it was, load it. If it wasn't, show the language picker
    */
   function onOfflineStorageReady() {
-    var storedLanguage = Adapt.offlineStorage.get('lang');
+    const urlParams = new URLSearchParams(window.location.search);
```

Comment:
sadly IE11 doesn't have support for `URLSearchParams`

Reply:
IE11 :(